packages/block-editor/src/components/block-lock/style.scss

```diff
@@ -25,24 +25,14 @@
 	}
 }
 .block-editor-block-lock-modal__checklist-item {
+	display: flex;
+	justify-content: space-between;
 	margin-bottom: 0;
 	padding: $grid-unit-15 0 $grid-unit-15 $grid-unit-40;
 
-	.components-base-control__field {
-		align-items: center;
-		display: flex;
-	}
-
-	.components-checkbox-control__label {
-		display: flex;
-		align-items: center;
-		justify-content: space-between;
-		flex-grow: 1;
-
-		svg {
-			margin-right: $grid-unit-15;
-			fill: $gray-900;
-		}
+	svg {
```

Comment:
> We need to be careful with element selectors like svg, as in this case where it's unintentionally applying to the checkmark svg as well 😄

Good catch! I usually have a good eye for that, so I'm surprised I missed that! 🤦‍♀️ 

> It might also be nice to add a `flex-grow: 0` in here so the icon doesn't shrink when the checkbox text is long.

I might be missing something, but I'm having trouble getting `flex-grow: 0` to work in that scenario. Would it be a bad idea to set a `min-width` on the icon instead?

Reply:
I think I meant flex-shrink 🙈 Does that work?